src/main/scala/com/gu/datalakealerts/Features.scala

```diff
@@ -138,4 +138,61 @@ object Features {
       }
     }
   }
+  case object OlgilBanner extends Feature {
```

Comment:
This [line](https://github.com/guardian/data-lake-alerts/pull/32/files#diff-ac7fae1faa0a8cd922c4949a768416b5R25) means that [we will automatically schedule monitoring for both platforms unless otherwise specified](https://github.com/guardian/data-lake-alerts/blob/master/src/main/scala/com/gu/datalakealerts/SchedulerLambda.scala#L29-L31). 

Given that attempting to monitor banner impressions on Android is going to throw an exception, we should add this line to both features:

```
val platformsToMonitor = List(iOS)
```

Reply:
Once you've added this line you should be able to confirm that Android Olgil/Braze Banner monitoring is not being scheduled anymore by running: https://github.com/guardian/data-lake-alerts#monitoring-schedule